WebDriverAgentLib/Categories/XCUIElement+FBWebDriverAttributes.m

```diff
@@ -238,6 +239,37 @@ - (BOOL)isWDHittable
   return nil == result ? NO : result.hittable;
 }
 
+/*! Whether the element is truly hittable based on XCUIElement.hittable */
+- (BOOL)isWDNativeHittable
```

Comment:
this will be a serious hit to the performance as it would try to resolve each element in the tree. Consider checking `fb_filterDescendantsWithSnapshots: onlyChildren:` for more details on how to match a snaphot to an element instance.